tests/unit/plugins/janitor/test_authority.py

```diff
@@ -162,6 +162,84 @@ def _create_fake(klass, *args, **kwargs):
         'rrsets', len(expected_rrsets), context=context)])
 
 
+@pytest.mark.asyncio
+async def test_whitelist_project_publishes_msg(mocker, authority_config,
+                                               get_gce_client, create_mock_coro,
+                                               instance_data, metrics):
```

Comment:
This also looks like a very thorough test (I haven't gotten all the way through it yet), but for such a small change I think it would be acceptable to do a unit test on the `_get_projects` method. For me it would be more clear to see what was changed and what to expect from the new behaviour.